src/core/agent/services/identifierService.ts

```diff
@@ -375,6 +377,86 @@ class IdentifierService extends AgentService {
     });
     return multisigId;
   }
+
+  async rotateMultisig(ourIdentifier: string): Promise<string | undefined> {
```

Comment:
Are we able to update the threshold of a multi-sig?

Reply:
Are you mean that threshold ? ![image](https://github.com/cardano-foundation/cf-identity-wallet/assets/142210850/97c03f01-7071-42a0-a1d7-e435e26d0ccd)

Reply:
yes

Reply:
I am not sure how to update it, but I try rotating the multisig, it is not changed

Reply:
I know, but I want to allow this to update it. Can you check in the Signify library on how to set it?

Reply:
For now, I don't see how to update it in Signify. May be we can create another story to research it in the future.